config/analytics.yml

```diff
@@ -6,6 +6,7 @@ analytics:
     analytics_id: <%= ENV['GOOGLE_ANALYTICS_ID'] %>
     app_name: <%= ENV['GOOGLE_OAUTH_APP_NAME'] %>
     app_version: <%= ENV['GOOGLE_OAUTH_APP_VERSION'] %>
-    privkey_path: <%= ENV['GOOGLE_OAUTH_PRIVATE_KEY_PATH'] %>
+    privkey_value: <%= ENV['GOOGLE_OAUTH_PRIVATE_KEY_VALUE'] %>
```

Comment:
If you take a look at the worker, the way this works on a deployment we need the path to be setup and not the value.

Reply:
hmm. I believe I should delete that then since we won't have analytics related env variables. that shouldn't cause a problem right?

Reply:
I am not sure, did you have issue navigating around in the dashboard after a dc down -v and a dc build?

I know I had an issue with it off main, and altered the worker script to add the `ENV['HYRAX_ANALYTICS']` to the if clause so the worker honors the flipper:

```yaml
if ENV['HYRAX_ANALYTICS'] && ENV['GOOGLE_OAUTH_PRIVATE_KEY_VALUE'] && !ENV['GOOGLE_OAUTH_PRIVATE_KEY_VALUE'].empty?
  %x{echo "$GOOGLE_OAUTH_PRIVATE_KEY_VALUE" | base64 -d > prod-cred.p12}
end
```

Reply:
I'll test this.

Reply:
``` bash
docker compose down -v
docker compose build
docker compose up -d && docker compose exec web sh
  bundle exec rails s -b 0.0.0.0 -p 3000
```

after the above, the app still loaded for me. I'll do a build with no cache overnight to be doubly sure though.

Reply:
I made the changes in commit [b03fe08](https://github.com/scientist-softserv/palni-palci/pull/479/commits/b03fe08c2477d70b9daaae931ff7871118d75c7d), ran `dc down -v` and `dc build --no-cache`. that failed because of a wip file I had introduced. I removed that file and ran `dc build` again, which passed. starting up the app with `dc up` shows that the web and worker containers have started (in my docker dashboard). however, I see no logs for either in my terminal or in the dashboard. see below. I also only get the dory "http 502" error in my browser. I've confirmed dory is running though.

<details>
<summary>dashboard and browser</summary>

<img width="874" alt="image" src="https://github.com/scientist-softserv/palni-palci/assets/29032869/a8b944cc-7263-4dac-bf7d-d62e8bf09448">

![image](https://github.com/scientist-softserv/palni-palci/assets/29032869/509b5099-7c8f-494c-9ac3-80f6c8566f76)
</details>

I don't understand why this is. the values I removed from the web and worker bin files are no longer used or referenced in this branch so it shouldn't cause a problem.

as info: I typically start up the app in a detached state, then manually start the server myself. (as noted in my above comment). so idk if the `dc up` command is also problematic on `main` for me.

Reply:
bringing up the app in a detached state and manually starting the server, works.

Reply:
april tested this locally and it worked for her so I deployed this branch to the staging env. it worked! no analytics are set by default, as expected.

![image](https://github.com/scientist-softserv/palni-palci/assets/29032869/e42bb068-3b79-463e-8ae8-0adc372bf53e)